config/bundles/bluima.preprocessing_0.1.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Sure. But should I also rename it to `preprocessing` only? And should I move it to `config/bundles/bluima` too?